src/jmcomic/jm_plugin.py

```
@@ -799,6 +799,91 @@ def ensure_make_pdf_dir(pdf_dir: str):
+        for img in images:
+            long_img.paste(img, (0, y_offset))
+            y_offset += img.height
+
+        long_img.save(long_img_path)
+        [img.close() for img in images]
+
+        return img_paths
+
+    @staticmethod
+    def get_img_dir(img_dir: str | None) -> str:
+        img_dir = fix_filepath(img_dir or os.getcwd())
```

Ruff

`fix_filepath` may be undefined, or defined from star imports

(F405)